winterfell/controllers/authController.js

```diff
@@ -41,17 +42,17 @@ module.exports = function (app) {
         var callbacks = {
           onError: function(errorCode, errorMsg) {
             console.log('[authSignUp] ' + errorCode + ' Error creating user: ' + errorMsg);
-            res.sendStatus(errorCode, errorMsg);
+            res.status(errorCode).send({error: errorMsg});
           },
           onSuccess: function(user) {
             console.log('[authSignUp] Successfully created user ' + user.externalId);
             var extUserId = user.externalId;
-            res.status(200).send({userId: extUserId, redirect: '/'});
+            res.status(http.CODE_SUCCESS).send({userId: extUserId, redirect: '/'});
```

Comment:
I think its a little confusing to have nonstandard names for the HTTP status codes. Fine with having constants for them, but if we do I think the constant for 200 should be name "OK" etc.